webui/src/pages/repositories/wizard/spark_config_step.jsx

```diff
@@ -44,6 +45,9 @@ export const SparkConfigStep = ({onComplete=()=>{}}) => {
     }
     return (
         <Box sx={{display: 'flex', flexDirection: 'column', justifyContent: 'space-between'}}>
+            <Box sx={{width: '100%'}}>
+                <Alert variant="warning" className="mt-3">Copy the Secret Access Key and store it somewhere safe. You will not be able to access it again.</Alert>
```

Comment:
nit; Consider keeping the text more consistent with https://github.com/treeverse/lakeFS/blob/33530b18aa599debe96cbb697f8da5aa5960f59d/webui/src/pages/setup/index.jsx#L73

Reply:
If we make them consistent I vote for the new version.
Would change "Secret Access Key" to "credentials" though.